src/shared/backgroundScripts/polyfill.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Polyfill the browser object to be compatible with both Chrome MV3 and Firefox MV2.
+ */
+export function polyfillBrowser() {
+  globalThis.browser ??= globalThis.chrome;
+  globalThis.browser.action ??= globalThis.browser.browserAction;
```

Comment:
Is there a reason to prefix everything with `globalThis.` here? It is only needed at `globalThis.browser ?==`, to assign the `browser` global.